src/check.pug

```diff
@@ -69,7 +70,9 @@ html(lang='it')
                                         button.btn.btn-primary.flex-grow-1(type="submit", data-lang="cta.continue") Paga 
                                             span.amount(id="check__totamount__button") &nbsp;
                                     .text-center
-                                        a.d-block.font-weight-bold.text-decoration-none.py-2(href="#",data-modal="#modal-checkwhypsp", data-modal-buttons="close") Perchè vedo Intesa San Paolo?
+                                        a.d-block.font-weight-bold.text-decoration-none.py-2(href="#",data-modal="#modal-checkwhypsp", data-modal-buttons="close") Perchè vedo&nbsp;
+                                            span(id="check__pspbankname") Intesa Sanpaolo
```

Comment:
@vito80ba `Intesa San Paolo` should be never shown, since we have the bank name in `sessionStorage`. What do you think about putting an empty string here?